### PR TITLE
Feat/add bearer auth

### PR DESCRIPTION
**Pull request by kyle-thedelta**
To be merged after https://github.com/reviopay/swagger-ui/pull/1

### Description

Added bearer auth functionality for the "try-it-out"

### Motivation and Context

Easier client integration

### How Has This Been Tested?

Local html render
Screenshot 2022-08-24 at 13 49 17

When trying "try-it-out" the following is returned:
Screenshot 2022-08-24 at 13 44 51
Which means its working correctly (the CORS is due to it not being from swagger.gate.revio.co.za)

### My PR contains... 

- [x] Features (non-breaking change which adds functionality)

### My changes...
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.

### Automated tests
- [x] My changes can not or do not need to be tested.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -11,6 +11,8 @@
   </head>
 
   <body>
+
+    <center><input style="position:relative; top:20px; box-shadow: 10px 10px #71D60A; border-width: 3px; border-color: red;" id="bearer-code-input" size="50" type="text" placeholder="Enter Bearer Token here" value="Enter bearer token here" /></center>
```
Comment (benatdelta) [inline dist/index.html]: nitpick: change the id to `bearer-token-input`?

Comment (benatdelta) [inline dist/index.html:15]: nitpick: change id to `bearer-token-input`?

**Comment by benatdelta** [comment]
What is the reason for adding the entire swagger doc again in dist/?

**Comment by kyle-thedelta** [comment]
> What is the reason for adding the entire swagger doc again in dist/?

I use a html previewer to see how the page looks as i make changes, so having the swagger here makes it easy.  
I was wondering if we should combine these 2 repos somehow (swagger-ui + revio-spell-swagger) as we need both to deploy the solution

**Pull request merged**